docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -1,25 +1,430 @@
+      Get:4 https://download.bareos.com/bareos/release/21/Debian_11  InRelease [1861 B]
+      Get:5 https://download.bareos.com/bareos/release/21/Debian_11  Sources [5660 B]
+      Get:6 https://download.bareos.com/bareos/release/21/Debian_11  Packages [36.0 kB]
+      Fetched 43.5 kB in 1s (42.3 kB/s)
+      Reading package lists... Done
+      Building dependency tree... Done
+      Reading state information... Done
+      15 packages can be upgraded. Run 'apt list --upgradable' to see them.
+   root@host:~# apt upgrade
+      Reading package lists... Done
+      Building dependency tree... Done
+      Reading state information... Done
+      Calculating upgrade... Done
+      The following packages will be upgraded:
+      bareos bareos-bconsole bareos-client bareos-common bareos-database-common bareos-database-postgresql bareos-database-tools bareos-director bareos-filedaemon bareos-storage bareos-tools libgssapi-krb5-2 libk5crypto3 libkrb5-3
+      libkrb5support0
+      15 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
+      Need to get 2557 kB of archives.
+      After this operation, 114 kB of additional disk space will be used.
+      Do you want to continue? [Y/n] Y
+
+.. index::
+   single: update minor bugfix; FreeBSD
+
+.. code-block:: shell-session
+   :caption: Shell example command to update Bareos on FreeBSD
+
+   root@host:~# pkg update --repository Bareos
+   root@host:~# pkg upgrade --repository Bareos
+      Updating Bareos repository catalogue...
+      Bareos repository is up to date.
+      All repositories are up to date.
+      Checking for upgrades (8 candidates): 100%
+      Processing candidates (8 candidates): 100%
+      The following 8 package(s) will be affected (of 0 checked):
+      Installed packages to be UPGRADED:
+            bareos.com-bconsole: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-common: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-database-common: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-database-postgresql: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-database-tools: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-director: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-filedaemon: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-storage: 21.0.0 -> 21.1.5 [Bareos]
+      Number of packages to be upgraded: 8
+      1 MiB to be downloaded.
+      Proceed with this action? [y/N]: y
+
+
+.. _section-UpdatePostChecks:
+
+Post update checks
+~~~~~~~~~~~~~~~~~~
+
+.. index::
+   single: Update post checks
+
+After the update, it is recommended to check if any new warnings are raised when starting the daemon.
+
+To do so you can use the `-t` flag:
+
+.. code-block:: shell-session
+   :caption: Shell example command to check bareos configuration
+
+   root@host:~# bareos-fd -t -u root -g bareos
+   root@host:~# bareos-sd -t -u bareos -g bareos
+   root@host:~# bareos-dir -t -u bareos -g bareos
```

Comment:
```suggestion
   root@host:~# su - bareos -s /bin/sh -c "bareos-dir -t"
   root@host:~# su - bareos -s /bin/sh -c "bareos-sd -t"
   root@host:~# bareos-fd -t
```
I always use these commands. Don't know, what is preferable.

Reply:
don't know which one it the best, especially to detect non accessible config file or working directory for example.